__tests__/app.test.js

```diff
@@ -37,26 +33,43 @@ describe("App", () => {
           expect(topicsArr.length).toBe(3);
         });
     });
-    test("should return 200 status and array of topic objects which contain 'slug' and 'description' properties", () => {
+    test("should return array of topic objects which contain 'slug' and 'description' properties", () => {
       return request(app)
         .get("/api/topics")
         .expect(200)
         .then((response) => {
           const body = response.body;
-          const topicsObj = body.topics[0];
-          expect(topicsObj).toHaveProperty("slug");
-          expect(topicsObj).toHaveProperty("description");
+          const topicsArr = body.topics;
+          for (const topicObj in topicsArr) {
```

Comment:
I think this bit of code is going to cause you a false positive as you are returning on line 44. This makes me think that you are not getting to line 47 which means that this test will pass as no assertion is ran. 

This should be refactored to a forEach and some consideration should be taken when using a for in loop with an array, which typically is something that is not done in JS. For in loops are designed to be used with object literals. 

Please change this to a forEach that can be called on each element of the topicsArr